docs/DeveloperGuide.md

```diff
@@ -258,61 +253,35 @@ Key Consideration: How to implement the relationship between `Event` and `Person
     * Additional overhead as new class(es) will have to be created and tested.
     * If implemented using a `EventPersonAssociation` list, will not be very efficient as well when making queries/changes,especially if there are a large number of associations in the list
 
-#### 2.2 Show event
-
-#### Implementation
-
-The following is a detailed explanation of the operations which take place when the `showevents` command is called
+#### 2.2 Implementing List event
 
-1. After successfully parsing of user input the `ShowEventCommand#exeucte(Model model)` method is called.
-2. Since all events are stored in a `FilteredList`, a `PREDICATE_SHOW_ALL_EVENTS` is passed to the list so that it now contains all the events in the addressbook. The `PREDICATE_SHOW_EVENTS` essentially returns `true` for every entry in the `FilteredList` i.e no item is being filtered out, thus all the events in the addressbook can be accessed through the list.
-3. The GUI contains a pointer to the `FilteredList` present in the `ModelManager`, thus it is able to retrieve all the events present in the addressbook and represent them as a `EventCard` class which is a JavaFX UI feature which visually depicts the event's name, date and time, description and the names of any friends associated with the event.
-4. The `MainWindow` class receives the `CommandResult` after `ShowEventsCommand::execute` is done executing, a boolean `event` within `CommandResult` decides whether `MainWindow::changeInterface` is called which changes the tab from `Friends` tab to `Events` or vice versa.
+Key Consideration: How to implement `showevents` when there exists multiple tabs and other `show` commands such as `listfriends` and `showinsights`.
 
-The following sequence diagram summarizes what happens when a user executes the `ShowEventsCommand`
+The following sequence diagram summarizes what happens when a user executes the `ShowEventsCommand`, the sequence diagram is similar for the other types of `show` commands which exist in Amigos.
 
 ![ShowEventSequenceDiagram.png](images\ShowEventsSequenceDiagram.png)
```

Comment:
not sure if its just me but can't seem to view the image on the DG website and intellij!  same for the other images `eventsTab and friendsTab` as well! I think can try changing it to`images/ShowEventsSequenceDiagram.png`

Reply:
Fixed